app/jobs/dhis2/bangladesh_exporter_job.rb

```diff
@@ -0,0 +1,23 @@
+module Dhis2
+  class BangladeshExporterJob < Dhis2ExporterJob
+    private
+
+    def facility_data_for_period(region, period)
+      {
+        htn_cumulative_assigned: PatientStates::Hypertension::CumulativeAssignedPatientsQuery.new(region, period).call.count,
```

Comment:
@govindkrjoshi Thinking of changing all of these query methods into class methods because they're a pain to test and also don't seem necessary. What does it mean to have a CumulativeAssignedPatientsQuery instance